examples/fem_system/fem_system_ex3/fem_system_ex3.C

```diff
@@ -57,15 +57,14 @@ int main (int argc, char ** argv)
   LibMeshInit init (argc, argv);
 
   // Parse the input file
-  GetPot infile("fem_system_ex3.in");
+  GetPot cl(argc,argv);
 
```

Comment:
Must this be either-or?  I've been moving example after example to a "read input file, then override from command line" model, which is really easy with GetPot and seems to offer the best of both worlds.